spec/savon/options_spec.rb

```diff
@@ -7,6 +7,16 @@
 
 RSpec.describe "Options" do
 
+  shared_examples(:deprecation) do
+    option = self.superclass.description[/:\w+/][1..-1].to_sym
```

Comment:
This is too clever, just pass the expected option as an argument instead. A spec shouldn't change behavior because you change the description